tsconfig.json

```diff
@@ -2,8 +2,8 @@
 	"extends": "@sindresorhus/tsconfig",
 	"compilerOptions": {
 		"target": "esnext",
-		"module": "es2015",
```

Comment:
I think this will increase the bundle size because ModuleConcatenationPlugin will stop working